tests/phpunit/integration/tests/Media/Media_Source_Taxonomy.php

```diff
@@ -53,16 +53,20 @@ public function test_register() {
 	 * @covers ::register_taxonomy
 	 */
 	public function test_register_taxonomy() {
-		$media = new \Google\Web_Stories\Media\Media_Source_Taxonomy();
+		$media_source = new \Google\Web_Stories\Media\Media_Source_Taxonomy();
+
+		$media = new \Google\Web_Stories\Media\Media_Source_Taxonomy( $media_source );
 		$this->call_private_method( $media, 'register_taxonomy' );
 
-		$this->assertTrue( taxonomy_exists( \Google\Web_Stories\Media\Media_Source_Taxonomy::TAXONOMY_SLUG ) );
+		$this->assertTrue( taxonomy_exists( $media_source->get_taxonomy_slug() ) );
```

Comment:
Why instantiate this class twice?

Also, the constructor doesn't take no args I think.

Reply:
Copy and paste 🤦

Reply:
Fixed